.github/workflows/preview.yaml

```
@@ -48,63 +147,79 @@ jobs:
+
+      - name: Download latest artifact
+        uses: actions/download-artifact@v4
         with:
-          node-version: 18
-          cache: npm
-
-      - name: Install dependencies
-        run: npm ci
+          name: cypress-snapshots
+          path: cypress/snapshots
+          filter: latest
```

actionlint (1.7.4)

input "filter" is not defined in action "actions/download-artifact@v4". available inputs are "github-token", "merge-multiple", "name", "path", "pattern", "repository", "run-id"

(action)